mapreduce.go

```diff
@@ -135,7 +135,7 @@ type MapReduceJob struct {
 }
 
 func Run(c context.Context, ds appwrap.Datastore, job MapReduceJob) (int64, error) {
-	log := appwrap.NewAppengineLogging(c)
```

Comment:
:( this will break our unit tests since it panics if it's not running in a real context.  I think a slightly more difficult (not much) but better change would be to make `appwrap.Logging` an argument to `Run`